src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -783,6 +783,13 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 				case IntrinsicId.Type_GetType: {
 						reflectionContext.AnalyzingPattern ();
 
+						var parameters = calledMethod.Parameters;
+						if ((parameters.Count == 3 && methodParams[2].Kind != ValueNodeKind.MergePoint && (methodParams[2].AsConstInt () == null || methodParams[2].AsConstInt () != 0)) ||
+							(parameters.Count == 5 && (methodParams[4].AsConstInt () == null || methodParams[4].AsConstInt () != 0))) {
+							reflectionContext.RecordUnrecognizedPattern ($"Reflection call '{calledMethod.GetDisplayName ()}' inside " +
+								$"'{callingMethodDefinition.GetDisplayName ()}' is trying to make a case insensitive lookup which is not supported by the linker");
```

Comment:
After you merge with master please create a new warning code for this, add it to the doc and so on.